.travis.yml

```diff
@@ -16,16 +16,11 @@ install:
 
 before_script:
   - make doc
-  - if [[ `git status --porcelain` ]]; then
-      echo "'make doc' was not run before commiting";
-      exit 1;
-    fi
+  - git status --porcelain || (echo "'make doc' was not run before commiting" && return 1)
   - go test ./...
 
 script:
-  - if [ "${TRAVIS_TAG::1}" = "v" ]; then
-      curl -sL https://git.io/goreleaser | bash;
-    fi
+  - '[[ $TRAVIS_TAG == v* && ! $TRAVIS_TAG =~ test ]] || return 0 && curl -sL https://git.io/goreleaser | bash'
```

Comment:
Parentheses after `return 0` would be clearer

Reply:
Oops I meant parentheses surrounding 
```
[[ $TRAVIS_TAG == v* && ! $TRAVIS_TAG =~ test ]] || return 0
```

Reply:
I think it would be a bad idea ... That would result in always executing the goreleaser process.

My goal was to exclude building a binary version when test is part of the tag, but since the project is also used as a library, that makes the library available without building & releasing a new version.

Reply:
How about
```
'([[ $TRAVIS_TAG == v* && ! $TRAVIS_TAG =~ test ]] && curl -sL https://git.io/goreleaser | bash) || return 0'
```

It's the operator priority that is confusing